pkg/backend/watch.go

```diff
@@ -35,8 +35,8 @@ import (
 func Watch(ctx context.Context, b Backend, stack Stack, op UpdateOperation, apply Applier) result.Result {
 
 	opts := ApplierOptions{
-		DryRun:   false,
-		ShowLink: false,
+		DryRun:      false,
+		CanShowLink: false,
```

Comment:
I’m not sure we need to change the name of this.

Reply:
Agree, I thought it could be a bit confusing since suppressPermalink and showLink have conflicting meanings

I.e showLink represents having the ability to produce a permalink whilst suppressPermalink represents whether it should actually show the permalink

Happy to change it back, though!